### PR TITLE
fix: the bug for the log in/sign up form

### PR DESCRIPTION
**Pull request by RuipuCui**
Added missing css code of login/sign up form
removed unnecessary console log statement
also fixed some typos

### CODE REVIEW

File: itp-front/src/components/Goal5LearningBanner.vue
Comment (Leshangpang) [inline itp-front/src/components/Goal5LearningBanner.vue:1]: Can you correct the spelling for Sustainable and Equality.

**Review by Leshangpang — CHANGES_REQUESTED** [review]
Correct the spelling in HomePageBanner

**Review by Leshangpang — APPROVED** [review]
Nice to see that the warning is now removed.

**Review by YuetongLiiii — APPROVED** [review]
The function is fixed now! good job!

**Pull request merged**